src/Auth.php

```diff
@@ -75,8 +81,23 @@ public static function login($email, $password, $challenge = '', $code = '') {
             ]);
         }
 
+        $client = new Client(['cookies' => true]);
+
+        $dataToken = FortniteClient::sendUnrealXSRFClientPostRequest($client);
+
+        $data = FortniteClient::sendUnrealClientLoginRequestPostRequest($client, $dataToken, $email, $password);
```

Comment:
Why do you need this if you don't use it anywhere?

Reply:
Those 2 functions make requests to log in. It uses cookiejar to use cookies from previous requests.